addon/templates/components/polaris-data-table/cell.hbs

```diff
@@ -1,88 +1,52 @@
-{{#if presentational}}
+{{#if (or header fixed)}}
   {{#if header}}
     <th
-      aria-hidden="true"
-      role="presentation"
-      id={{cellElementId}}
+      {{!-- template-lint-disable --}}
+      data-test-data-table-cell
+      data-test-data-table-cell-header
+      data-test-data-table-cell-sorted="{{sorted}}"
```

Comment:
This doesn't get assigned a boolean value in the test environment if I use 

```hbs
data-test-data-table-cell-sorted={{sorted}}
```

which is why it's in quotes with the linter disabled for this element. Same with the footer test-selector at the bottom of this files. I'm good with just checking the Polaris classes for these two and dropping the helpers if that sounds more reasonable.

Reply:
I'm OK with this if it works 👍

Reply:
@vladucu we have tests that test that the correct number of cells were rendered, and we also have tests that test headers by themselves. That's why we have multiple selectors because not all cells are headers, etc.

Reply:
not sure I understand tbh or how this would help with this case (if I understand correctly lol)
if you want to easier select header cells/etc....just add a selector in the main data-table component so you can scope the selector to it....do you think something like that would work?

Reply:
Yeah that works too, I was just trying to put selectors on the elements we were actually selecting vs. using them as a scope and still using `th` or `td` in the selection path. I'll update this.